config/dashboards/dashboards_config.go

```diff
@@ -128,6 +130,18 @@ func GetBuiltInMonitoringDashboards() MonitoringDashboardsList {
 	}
 }
 
+func GetNamespaceMonitoringDashboards(annotations map[string]string) MonitoringDashboardsList {
```

Comment:
Since this is a public API, should add a godoc comment to this function:

```suggestion
// GetNamespaceMonitoringDashboards will examine the given namespace annotations and return any dashboard yaml found.
func GetNamespaceMonitoringDashboards(annotations map[string]string) MonitoringDashboardsList {
```